lib_board_support/src/boards/xk_audio_316_mc_ab/xk_audio_316_mc_ab_board.xc

```diff
@@ -15,26 +15,24 @@ extern "C" {
     #include "sw_pll.h"
```

Comment:
Note to self, I wonder if the extern "C" can be put in lib_sw_pll instead using an ifdef __XC__